FilmFestivalLoader/IFFR/parse_iffr_html.py

```diff
@@ -297,7 +552,7 @@ def read_url(self, url):
         return html
 
 
-class FilmPageParser(HTMLParser):
+class OLdFilmPageParser(HTMLParser):
```

Comment:
Maybe I need to steel parts of it when loading all films instead of a small test set.
I'll drop it in the final version, where the screened film description will also be loaded.